documentation/monitoring.md

```diff
@@ -0,0 +1,58 @@
+# Monitoring and Logging
+
+## Logging
+
+### Application logs
+
+- [Sidekiq](https://teaching-vacancies.service.gov.uk/sidekiq)
+- [Check the Rollbar dashboard](https://rollbar.com/dfe/teacher-vacancies/) to see if any Errors are being logged.
+- [Check the Papertrail event log](https://papertrailapp.com/events) to see in real-time if any Errors are being logged.
+
+### Deployment logs
+
+- [Check the GitHub Deploy workflow](https://github.com/DFE-Digital/teaching-vacancies/actions?query=workflow%3ADeploy) to see recent deployments.
+
+### Cloudfront
+
+- [Cloudfront log dashboard](https://console.aws.amazon.com/cloudfront/v2/home#/logs/E37BGF9R499RSE). When standard logging is enabled, this goes to a [logging S3 bucket](https://console.aws.amazon.com/s3/buckets/530003481352-tv-cloudfront-logs?region=eu-west-2&prefix=production/&showversions=false) recording logs in gzipped format
+
+## Monitoring, Metrics and Usage
+
+### Grafana, Prometheus, and AlertManager
+
+For several infrastructure components on Gov.UK PaaS, we use [Prometheus Exporters](https://prometheus.io/docs/instrumenting/exporters/) to:
+- fetch statistics from another, non-Prometheus system
+- turn those statistics into Prometheus metrics, using a client library
+- expose a `/metrics` URL, and have that URL display the system metrics (in the case of teaching vacancies, rather than appending `/metrics` to the root URL, it's instead at [https://paas-prometheus-exporter-teaching-vacancies.london.cloudapps.digital/metrics](https://paas-prometheus-exporter-teaching-vacancies.london.cloudapps.digital/metrics))
+
+The [Grafana dashboard](https://grafana-teaching-vacancies.london.cloudapps.digital/d/6Ac4lUWGk/teaching-vacancies-production?orgId=1&refresh=5s) has been set up with useful visualisations of metrics over time for:
```

Comment:
We could also mention the shared dashboards

Reply:
Thanks. Good spot. I've added the CF apps dashboard as the most-useful.